qcoverage/src/org/qcmg/coverage/Options.java

```diff
@@ -54,69 +36,53 @@
-				.withRequiredArg().ofType(String.class).describedAs(
-						"expression");
-		parser.acceptsAll(asList("n"), NUMBER_THREADS_DESCRIPTION)
-				.withRequiredArg().ofType(Integer.class).describedAs(
-						"number of worker threads");
-		parser
-				.acceptsAll(asList("per-feature"),
-						PER_FEATURE_OPTION_DESCRIPTION);
-		parser.acceptsAll(asList("v", "V", "version"), VERSION_DESCRIPTION);
-		parser.accepts("log", LOG_OPTION_DESCRIPTION).withRequiredArg().ofType(
-				String.class);
-		parser.accepts("loglevel", LOG_LEVEL_OPTION_DESCRIPTION)
-				.withRequiredArg().ofType(String.class);
-		parser.accepts("validation", VALIDATION_STRINGENCY_OPTION_DESCRIPTION)
-				.withRequiredArg().ofType(String.class); 
+		parser.accepts("help", HELP_DESCRIPTION);	
+		parser.accepts("version", VERSION_DESCRIPTION);
+		parser.accepts("log", LOG_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);
+		parser.accepts("loglevel", LOG_LEVEL_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);
 		
+		parser.accepts("output", OUTPUT_DESCRIPTION).withRequiredArg().ofType(String.class);	
+		parser.accepts("output-format", OUTPUT_FORMAT_DESCRIPTION).withRequiredArg().ofType(String.class);
+				
+		parser.accepts("input-bam", INPUT_BAM_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);				
+		parser.accepts("input-bai", INPUT_BAI_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);		
+		parser.accepts("input-gff3", INPUT_GFF3_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);				 
+		parser.accepts("type", TYPE_OPTION_DESCRIPTION)	.withRequiredArg().ofType(String.class);
+		parser.accepts("query", QUERY_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);
+		parser.accepts("thread", NUMBER_THREADS_DESCRIPTION).withRequiredArg().ofType(Integer.class);
+		parser.accepts("per-feature", PER_FEATURE_OPTION_DESCRIPTION);						
+		parser.accepts("validation", VALIDATION_STRINGENCY_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class); 
 		
-		//segmenter options
-		parser.accepts("segmenter", SEGMENTER_OPTION_DESCRIPTION);
-		parser.accepts("infile", INPUT_DESCRIPTION).withRequiredArg().ofType(String.class).describedAs(
-				"inputfile");
-		parser.acceptsAll(asList("outfile"), Messages
-				.getMessage("OUTFILE_DESCRIPTION")).withRequiredArg().ofType(String.class).describedAs(
-				"outputfile");
-		parser.acceptsAll(asList("f", "feature"), Messages
-				.getMessage("FEATURE_DESCRIPTION")).withRequiredArg().ofType(String.class).describedAs(
-				"feature");
-		parser.accepts("merge", Messages.getMessage("MERGE_OPTION_DESCRIPTION"));
-		parser.accepts("fill", Messages.getMessage("FILL_OPTION_DESCRIPTION"));
-		parser.accepts("bounds", Messages.getMessage("BOUNDS_OPTION_DESCRIPTION")).withRequiredArg().ofType(String.class).describedAs(
-				"name of bounds");
-		parser.accepts("reference", Messages.getMessage("REFERENCE_OPTION_DESCRIPTION")).withRequiredArg().ofType(String.class).describedAs(
-				"reference");
+
```

Comment:
is the `segmenter` option used? If yes, keep code in place (not commented out), if no longer required, delete (maybe check with JP)

Reply:
The segmenter option is deprecated, it is not used anywhere.  As we often do is to mark it as deprecated first, and then delete it on next release